10-slots/01-UiPaginationView/components/UiPaginationView.vue

```diff
@@ -26,6 +30,20 @@ export default {
       required: true,
     },
   },
+
+  computed: {
+    localItems() {
+      return this.items.map((it) => ({
+        id: --id,
+        item: it,
+      }));
+    },
```

Comment:
This is an example of a bad way to add a key. This computed `localItems` will be recreated with a new `id` on any `items` changes. It means, that on any list change, an entire rendered list will be re-created.

Reply:
In this component, we have to options:
1. Ask the component's user to provide object property for `key` or function
2. Ask the component's user to manually set `key` in the template provided to the slot

I prefer the second one.